docs/metrics/getting-started-prometheus-grafana/README.md

```diff
@@ -26,7 +26,7 @@ Add a reference to [Prometheus
 Exporter](../../../src/OpenTelemetry.Exporter.Prometheus/README.md):
 
 ```sh
-dotnet add package OpenTelemetry.Exporter.Prometheus --version 1.2.0-rc2
+dotnet add package OpenTelemetry.Exporter.Prometheus --version 1.2.0
```

Comment:
```suggestion
dotnet add package --prerelease OpenTelemetry.Exporter.Prometheus
```

Prometheus skipped 1.2.0. It's not stable yet, so latest version is 1.3.0-beta.`

Reply:
https://www.nuget.org/packages/OpenTelemetry.Exporter.Prometheus